extensions/plugins/system/redcore/redcore.php

```diff
@@ -316,6 +316,18 @@ public function onBeforeCompileHead()
 				unset($doc->_scripts[JURI::root(true) . '/templates/' . $template . '/js/jui/bootstrap.js']);
 				unset($doc->_scripts[JURI::root(true) . '/templates/' . $template . '/js/jui/bootstrap.min.js']);
 			}
+
+			// Remove permission.js. Specially for Joomla 3.6
+			if ($isAdmin && version_compare(JVERSION, '3.6.0', '>='))
```

Comment:
It's not happen because we have a check here. So, this script just work on redCORE's extensions.

https://github.com/redCOMPONENT-COM/redCORE/blob/develop/extensions/plugins/system/redcore/redcore.php#L216